react-js/react.md

```diff
@@ -4,21 +4,22 @@
 8 Hours
 
 ### Prerequisites
-* [JavaScript I - VI](../javascript)
-* [jQuery Topic Outline](/web/jquery.md)
+* [HTML](../web/html.md)
+* [JavaScript 6 - OOP or above](../javascript/javascript-7-oop.md)
 
 ### Motivation
-I want to create interactive UIs more easily and with less code than jQuery.
+React.js is the most popular front-end framework for Web applications. 
```

Comment:
Popular isn't necessarily a reason to learn something, can we go with a value statement like:

"React makes it easy to update your HTML as data changes and build complex apps out of simple, reusable components."

Reply:
Okay.